README.md

```diff
@@ -1,2 +1,7 @@
 # knotx-handlebars-extension
 Custom Knot.x Handlebars Helpers
+
+How to install:
+* `mvn clean package`
+* Add `knotx-handlebars-extension-x.y.z.jar` to your knotx classpath.
+
```

Comment:
Maybe we should list here all available extensions?
E.g.

## Helpers

### ElementHelper
Get the n-th element from a json array.

**Usage**
```
{{element array n}}
```

**Example**
```
{
  locales: ["en_us", "pl_pl", "en_uk"]
}
```
```
{{element locales 2}} // "en_uk"

```

### Conditions checks
#### AND
...

### Math helpers
...

### Uri helpers
...